game/main.cpp

```diff
@@ -11,8 +11,12 @@
 int main(int argc, char** argv) {
   while (true) {
     spdlog::set_level(spdlog::level::debug);
+    auto game_logger = spdlog::basic_logger_mt("GOAL Runtime", "logs/runtime.log");
+    spdlog::set_default_logger(game_logger);
+    spdlog::flush_on(spdlog::level::info);
```

Comment:
This is great, we can change this if we ever want to see more or less logs for debugging.

Reply:
Excellent! We should be able to have individual log files for the compiler, decompiler, and runtime, or at least individual logger objects.